src/api/Visualization/types.ts

```diff
@@ -424,3 +424,8 @@ export interface CreateVisualizationCommentParams extends IDataSourceParams {
 export interface GetVisualizationCommentsParams extends IDataSourceParams, PaginationClause {
   visualizationId: number;
 }
+
+export interface DeleteVisualizationCommentParams extends IDataSourceParams {
+  commentId: number;
+  visualizationId: number;
```

Comment:
Unless there is something that i am missing, i don't think the `visualizationId` is necessary since the `commentId` will always be unique

Reply:
yeah it is not needed as the comment instance contain the id of the viz to which it belongs we can just getting from it.